android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/DenyMobileAuthConfirmationUseCase.kt

```diff
@@ -0,0 +1,28 @@
+package com.onegini.mobile.sdk.reactnative.clean.use_cases
+
+import com.facebook.react.bridge.Promise
+import com.onegini.mobile.sdk.reactnative.OneginiSDK
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors.MOBILE_AUTH_OTP_IS_DISABLED
+import com.onegini.mobile.sdk.reactnative.exception.OneginiWrapperErrors.MOBILE_AUTH_OTP_NOT_IN_PROGRESS
+import com.onegini.mobile.sdk.reactnative.handlers.mobileauthotp.MobileAuthOtpRequestHandler
+import javax.inject.Inject
+import javax.inject.Singleton
+
+
+@Singleton
+class DenyMobileAuthConfirmationUseCase @Inject constructor(
+    private val oneginiSDK: OneginiSDK,
+    private val mobileAuthOtpRequestHandler: MobileAuthOtpRequestHandler
+) {
+    operator fun invoke(promise: Promise) {
+        when (oneginiSDK.config.enableMobileAuthenticationOtp) {
```

Comment:
Use `if/else` in both cases here and below. If below one will feel too big extract into separate private method.